notifier.rst

```diff
@@ -553,6 +553,26 @@ and
 :class:`Symfony\\Component\\Notifier\\Notification\\EmailNotificationInterface`
 also exists to modify messages send to those channels.
 
+Development & Debugging
+-----------------------
```

Comment:
I would remove this title for now. We can always introduce a debug/dev section if we're documenting more dev features.

Reply:
I agree. I did this while merging. Thanks.